source/install/cloud-changelog.md

```diff
@@ -4,12 +4,51 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2022-04-27](#release-2022-04-27)
 - [Release 2022-04-13](#release-2022-04-13)
 - [Release 2022-03-30](#release-2022-03-30)
 - [Release 2022-03-16](#release-2022-03-16)
 - [Release 2022-03-08](#release-2022-03-08)
 - [Release 2022-03-02](#release-2022-03-02)
-- [Release 2022-02-16](#release-2022-02-16)
+
+## Release 2022-04-27
+
+### Improvements
+
+#### User Interface (UI)
+ - Added the channel members list to the right-hand side Channel Info modal.
+ - Added the ability to invite new users to a team from the **Add to channel** modal.
+ - To be able to download images and copy public links for images quicker, a copy URL and download buttons were added to image thumbnails.
+ - Added the ability to have one-character long channel names.
+
+#### Administration
+ - Added a new config cleanup job to regularly remove outdated entries from the database. The threshold for this setting can be adjusted with ``JobSettings.CleanupConfigThresholdDays``.
+ - Added a new API endpoint ``POST /api/v4/users/{user_id}/teams/{team_id}/threads/{thread_id}/set_unread/{post_id}`` to set a thread as unread by post id.
+ - Added ``fromWebhook`` property to the webapp plugin API.
+ - Added new API endpoints ``GET /api/v4/teams/:team_id/top/reactions`` and ``GET /api/v4/users/me/top/reactions`` to get top reactions for a team and user.
+ - Bumped Go to v1.18.
+
+#### Performance
+ - Improved the performance of ``GetTeamsUnreadForUser`` when Collapsed Reply Threads is enabled.
+ - Improved the performance of configuration read/writes if the configuration is stored on a database.
+
+#### Bug Fixes
+ - Fixed an issue where the ``UpdateUser`` API endpoint required a ``create_at`` field.
+ - Fixed an issue where permalinks to Direct and Group Message posts did not show a preview.
```

Comment:
Docs is moving away from capitalizing product features like Direct Messages or Group Messages.